CHANGELOG.md

```diff
@@ -10,6 +10,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 ## [0.3] - 2025-02-06
 ### Added
+- `_computed` internal cache flag attribute
```

Comment:
Please relocate it to the `Unreleased` section. It is currently in the wrong place.